libsolidity/codegen/ReturnInfo.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <libsolidity/codegen/ReturnInfo.h>
+
+#include <libsolidity/ast/Types.h>
+#include <libsolidity/ast/AST.h>
+
+using namespace solidity::frontend;
+using namespace solidity::langutil;
+
+ReturnInfo::ReturnInfo(EVMVersion const& _evmVersion, FunctionType const& _functionType):
+	dynamicReturnSize(false),
+	estimatedReturnSize(0)
```

Comment:
We don't need those.

Reply:
We still don't need these - they are member-initialized, doing it twice over is error-prone :-).